content/200-concepts/100-components/drivers.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Drivers'
+metaTitle: 'Drivers (Preview)'
+metaDescription: 'Learn how you can use Prisma with other database drivers using driver adapters (Preview)'
+preview: true
+tocDepth: 3
+---
+
+## Built-in drivers (default)
+
+One of Prisma Client's component's is the [Query Engine](./prisma-engines/query-engine) <span class="concept"></span>. The Query Engine transforms Prisma Client queries to SQL statements that are executed using the included database driver. By default, the Query Engine uses TCP connections to run a query against your database.
+
+<!-- TODO: add image here -->
+
+## Database drivers
+
+Prisma Client can connect and run queries against your database using other JavaScript database drivers using **driver adapters**. Adapters act as _translators_ between Prisma Client and the JavaScript database driver. Prisma will use the Query Engine to transform the Prisma Client query to SQL and run the generated SQL queries via the JavaScript database driver.
+
+To use this feature:
+
+1. Update the `previewFeatures` block in your schema to include the the `driverAdapter` preview feature:
+
+   ```prisma
+   generator client {
+     provider        = "prisma-client-js"
+     previewFeatures = ["driverAdapters"]
+   }
+   ```
+
+2. Generate Prisma Client:
+
+   ```sh
+   npx prisma generate
+   ```
+
+## Serverless drivers
```

Comment:
```suggestion
### Serverless drivers
```
Driver Adapters for Serveless drivers are the one currently supported kind of driver adapters.

Reply:
Maybe even:
```suggestion
### Driver Adapters for Serverless Drivers 
```